configs/roles/integrations.json

```diff
@@ -0,0 +1,16 @@
+{
+  "roles": [
+    {
+      "name": "Integrations administrator",
+      "description": "Perform any available operation against any Integrations resource.",
+      "system": true,
+      "platform_default": false,
+      "version": 1,
```

Comment:
The role versions actually need to be started at `"version": 2` due to some nuance in the way our seeds run.

Reply:
I had no clue :-)